manifest.yml

```diff
@@ -1,10 +1,11 @@
 name : "coreHTTP"
 version: "v1.0.0"
 description: |
-  "Client implementation of the HTTP/1.1 specification for embedded devices."
-license: "MIT"
-name : "http-parser"
-version: "v2.9.3"
-description: |
-  "This is a parser for HTTP messages written in C. It parses both requests and responses. The parser is designed to be used in performance HTTP applications. It does not make any syscalls nor allocations, it does not buffer data, it can be interrupted at anytime. Depending on your architecture, it only requires about 40 bytes of data per message stream (in a web server that is per connection)."
+  "Client implementation of the HTTP/1.1 specification for embedded devices.\n"
 license: "MIT"
+dependencies:
```

Comment:
Dependencies should be before license to be consistent with CSDK

Reply:
ok